test.sh

```diff
@@ -218,6 +218,9 @@ if [[ "${RUN[@]}" =~ "$STAGE" ]] ; then
   run_and_expect_silence codespell \
     --ignore-words=.codespell.ignore.txt \
     --skip=.git,.gocache,go.sum,go.mod,vendor,bin,*.pyc,*.pem,*.der,*.resp,*.req,*.csr,.codespell.ignore.txt,.*.swp
+  # Check test configs are formatted consistently
+  find test/config test/config-next -name \*.json -exec sh -c 'python3 -m json.tool "$1" "$1.tmp" && mv "$1.tmp" "$1"' shell {} \;
```

Comment:
Given that this is a Go repository, any chance I can talk you into passing the `--tab` flag to json.tool, so that the config files use tabs for semantic indentation like gofmt does?

Reply:
https://pkg.go.dev/cmd/gofmt@master
>  It uses tabs for indentation and blanks for alignment.

Reply:
The version of python3 that's in the docker containers right now doesn't support `--tab`.

Instead of gluing a somewhat complex `find` expression to python -m json.tool, a small standalone Python script could handle this instead (and then we can indent with whatever we want).   Do you think that's worth doing @aarongable?

I've got two hours left on my flight so I can probably whip that up :)

Reply:
OK I've rewritten as pure and using tabs for indent